src/opera/test/util/test_h5_utils.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+"""
+======================
+test_h5_utils.py
+======================
+
+Unit tests for the util/h5_utils.py module.
+
+"""
+
+import os
+import tempfile
+import unittest
+
+from opera.util.h5_utils import create_test_cslc_metadata_product
+from opera.util.h5_utils import create_test_disp_metadata_product
+from opera.util.h5_utils import create_test_rtc_metadata_product
+from opera.util.h5_utils import get_cslc_s1_product_metadata
+from opera.util.h5_utils import get_disp_s1_product_metadata
+from opera.util.h5_utils import get_rtc_s1_product_metadata
+
+
+def osr_is_available():
+    """
+    Helper function to check for a local installation of the Python bindings for
+    the Geospatial Data Abstraction Library (GDAL).
+    Used to skip tests that require GDAL if it is not available.
+    """
+    try:
+        from osgeo import osr  # noqa: F401
+        return True
+    except (ImportError, ModuleNotFoundError):
+        return False
+
+
+class H5UtilsTestCase(unittest.TestCase):
+    """Unit test H5 Utilities"""
+
+    # @skipIf(not osr_is_available(), reason="osgeo.osr is not installed on the local instance")
```

Comment:
Remove before merge

Reply:
I'm sorry, I don't understand what I'm supposed to remove.  Are you talking about the extra equal signs above and below the module name?

Reply:
the decorator I image, now that I look again.  I'll wait to hear from you.